autogpt_platform/backend/backend/blocks/llm.py

```diff
@@ -52,6 +52,10 @@ def AICredentialsField() -> AICredentials:
         description="API key for the LLM provider.",
         provider="llm",
```

Comment:
imo this needs to be a structure that shows the allowable providers, not its own provider. something like 

valid_providers = ['openai', 'grok', 'anthropic']
and be used in conjunction with discriminator always to have one of the providers be selected

Reply:
this is done! now `provider` can be a `list` (and is for AI blocks)